packages/edit-site/src/components/sidebar-navigation-screen-pattern/use-pattern-details.js

```diff
@@ -101,6 +70,60 @@ export default function usePatternDetails( postType, postId ) {
 		} );
 	}
 
+	const templatePartAreaLabels = {
+		header: __( 'Header' ),
+		footer: __( 'Footer' ),
+		sidebar: __( 'Sidebar' ),
+		uncategorized: __( 'General' ),
+	};
+
+	if ( postType === 'wp_template_part' ) {
+		details.push( {
+			label: __( 'Area' ),
+			value: templatePartAreaLabels[ record.area ],
+		} );
+	}
+
+	if (
+		postType === 'wp_template_part' &&
+		addedBy.text &&
+		! isAddedByActiveTheme
+	) {
+		details.push( {
+			label: __( 'Added by' ),
+			value: (
+				<>
+					<span className="edit-site-sidebar-navigation-screen-pattern__added-by-description-author">
+						{ addedBy.text }
+					</span>
+				</>
+			),
+		} );
+	}
+
+	if (
+		postType === 'wp_template_part' &&
+		addedBy.text &&
+		( record.origin === 'plugin' || record.has_theme_file === true )
+	) {
+		details.push( {
+			label: __( 'Customized' ),
+			value: (
+				<>
+					{ addedBy.isCustomized ? (
+						<span className="edit-site-sidebar-navigation-screen-pattern__added-by-description-customized">
+							{ _x( 'Yes', 'pattern' ) }
+						</span>
+					) : (
+						<span className="edit-site-sidebar-navigation-screen-pattern__added-by-description-customized">
+							{ _x( 'No', 'pattern' ) }
+						</span>
+					) }
+				</>
```

Comment:
This can be simplified to just the ternary operator, returning the appropriate span. In other words, we can drop the JSX fragment wrapper.